packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -342,67 +371,61 @@ export const Account = ({
             ))}
           </>
         ) : (
-          <li data-testid="" className="gradient-info-display">
+          <li
+            data-testid=""
```

Comment:
If we're not testing these, we can just remove the `data-testid` attributes set to empty strings.